pdns/dnsdistdist/m4/ax_python_module.m4

```diff
@@ -0,0 +1,56 @@
+# ===========================================================================
```

Comment:
Ahh, I wasn't sure if symlinks were safe here. I can symlink `pdns_check_python_venv.m4` too, since it's identical in all three locations (the module-specific tests were moved into each module's configure script).

Reply:
yes! symlinking the other m4s was implied :)